src/components/Disputes/DisputeDetail.js

```diff
@@ -35,6 +35,8 @@ const DisputeDetail = React.memo(function DisputeDetail({ match }) {
     requests,
   } = useDisputeLogic(disputeId)
 
+  const creatorAddress = dispute && dispute.subject && dispute.subject.id
```

Comment:
I think [we can use optional chaining](https://github.com/babel/babel/issues/10809) now! \o/

```suggestion
  const creatorAddress = dispute?.subject?.id
```